src/main/resources/config/application-dev.yml

```diff
@@ -163,9 +163,18 @@ application:
   frontend:
     sentry-project-id:
     google-analytics-project-id:
+    recaptcha-project-id:
+    recaptcha-site-key:
+    recaptcha-threshold:
+    recaptcha-enabled:
   aws:
     s3-access-key:
     s3-secret-key:
     s3-region:
   github-token:
   db-read-only: false # Certain endpoints will be disabled when readonly is set to true
+  recaptcha:
+    project-id:
+    site-key:
+    threshold:
+    enabled:
```

Comment:
threshold and enabled need to have default value, otherwise they need to be String.
We could set to false and 0.